Sources/Drain.swift

```diff
@@ -33,9 +34,6 @@ public final class Drain: DataRepresentable, Stream {
 
     public init(_ buffer: Data) {
         self.buffer = buffer
-        if buffer.bytes.isEmpty {
-            close()
-        }
```

Comment:
I think we should add the option to init open or closed. 

> `(bodyForReceiver.receiver as? Drain)?.open()`

This is ugly. :cry: